README.md

```diff
@@ -75,6 +75,44 @@ return array(
 )
 ```
 
+###Lumen Installation
+
+Begin by installing this package through Composer. Edit your project's composer.json file to require get-stream/stream-laravel.
+
+```
+"require": {
+    "get-stream/stream-laravel": "dev-master"
```

Comment:
please don't do that :-)

Just say: `composer require get-stream/stream-laravel`. Composer will figure out the best version constraint and add it to your composer.json file

Reply:
This will work if you tag a new version once merged, otherwise it will just choose the latest version (which will not include the LumenServiceProvider)

Reply:
Haha yeah :-) but I will tag a new version as soon as documentation is on point :-)